updater/reports/ReportRepositoryHistory.py

```diff
@@ -0,0 +1,48 @@
+from .ReportDaily import *
+
+# Lists the total number of repositories
+class ReportRepositoryHistory(ReportDaily):
```

Comment:
minor nit: I would prefer to use "Repo" in the code instead of "Repository" to improve code readability 😄

Reply:
I get your point. Personally, I don’t like using abbreviations at all, as I have encountered so many cases where abbreviations introduced unnecessary ambiguities.

For instance, I had to work on a piece of code that had classes suffixed `Impl`, which—surprise—didn’t stand for ”implementation” but “implication,” causing a major misunderstanding of the code.

Well, while ”repo” might not be misunderstood as something else, I made a practice of not abbreviating anything for no good reason. “PR” is an example that my intuition tells me could be misunderstood, as I’d immediately interpret it as “public relations” before thinking about “pull requests.”

However, it might be more important not to discuss this for too long (as I did with this comment, shame on me) but to keep things consistent within the code base (which isn’t the case right now). So maybe, we should clean up all the files at some point after making a uniform decision on how to name things (for instance, according to GitHub’s MySQL schemes or the GitHub API).

Reply:
In general I concur (I [love this post](https://twitter.com/collision/status/602950284864692224)). However, `Org`, `Repo`, and `PR` in the context of GitHub are pretty clear from my point of view.

Reply:
For the record: @larsxschneider and I agreed to consistently name entities as done be the [GitHub API](https://developer.github.com/v3/) in the future 😃.

There, pull requests are abbreviated `pulls`, repositories are `repos`, and organizations are `orgs`, for instance.

We’ll try to make this consistent soon!